xinference/client.py

```diff
@@ -58,10 +58,45 @@ def generate(
             Union["LlamaCppGenerateConfig", "PytorchGenerateConfig"]
         ] = None,
     ) -> Union["Completion", Iterator["CompletionChunk"]]:
+        """
+        Creates a completion for the provided prompt and parameters.
+
+        Parameters
+        ----------
+        prompt : str
+            The user's input.
+        generate_config: Optional["ChatglmCppGenerateConfig"]
+            Additional configurations for completion.
+
+        Returns
+        -------
+        Union["Completion", Iterator["CompletionChunk"]]
+            The Completion chunk of the model.
+            stream=True -> Iterator["CompletionChunk"]
```

Comment:
Explain `stream`.

Reply:
added explanation for both what happening to stream = true / stream = false in the latest version